swatch-system-conduit/src/main/spec/internal-organizations-sync-api-spec.yaml

```diff
@@ -0,0 +1,312 @@
+openapi: "3.0.2"
+info:
+  title: "internal-organizations-sync-api"
+  description: "REST interface for the internal-organization-sync- service. Please note any deprecated APIs. Our current deprecation policy is to keep deprecated APIs around for at least 6 months."
+  version: 1.0.0
+
+servers:
+  - url: /{PATH_PREFIX}/{APP_NAME}/v1
+    variables:
+      PATH_PREFIX:
+        default: api
+      APP_NAME:
+        default: rhsm-subscriptions
+  - url: https://{HOSTNAME}/{PATH_PREFIX}/{APP_NAME}/v1
+    variables:
+      HOSTNAME:
+        enum:
+          - ci.cloud.redhat.com
```

Comment:
@kahowell should we still include CI and QA in this enumeration?

Reply:
Is there a way to avoid specifying hostnames which can change in the API at all?

Reply:
Yeah, we let's just drop `servers` from the spec altogether (it's optional anyways).